File: src/java/main/commands/RichJson_file.java
package commands;

import core.RichJsonContext;
import core.RichJsonFileHelper;
import core.RichJsonHelper;
import core.RichJsonParser;

public class RichJson_file implements RichJsonCommand {
    @Override
    public Object execute(RichJsonParser parser, RichJsonContext context) {
        String fileName = RichJsonHelper.concatStrings((String) context.currentMember, ".json");
        return RichJsonFileHelper.readFile(fileName, true);
    }
}

File: src/c#/main/commands/RichJson_file.cs
using System;
using main.core;
using main.helper;

namespace main.commands
{
    public class RichJson_file
    {
        public object Execute(RichJsonParser parser, RichJsonContext context)
        {
            string fileName = context.CurrentMember + ".json";
            return RichJsonFileHelper.ReadFile(fileName, true);
        }
    }
}